wrappers/mongo/mongo_test.go

```diff
@@ -0,0 +1,120 @@
+package epsagonmongo
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/benweissmann/memongo"
+	"github.com/epsagon/epsagon-go/epsagon"
+	"github.com/epsagon/epsagon-go/protocol"
+	"github.com/epsagon/epsagon-go/tracer"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var mongoServerURI string
+var docsInserted int64 = 0
+
+func TestMongoWrapper(t *testing.T) {
+	mongoServer, err := memongo.Start("4.2.0")
```

Comment:
It would be better to setup the mongo DB at  each test, so tests won't be affected from other tests actions